src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -404,322 +409,302 @@ bool FlexPA::enclosesOnTrackPlanarAccess(
         1003,
         "enclosesPlanarAccess: layer is neither vertical or horizontal");
   }
-  const auto& tracks = trackCoords_[layerNum];
-  const auto lowTrack = tracks.lower_bound(low);
-  if (lowTrack == tracks.end()) {
+  const auto& tracks = track_coords_[layer_num];
+  const auto low_track = tracks.lower_bound(low);
+  if (low_track == tracks.end()) {
     logger_->error(DRT, 1004, "enclosesPlanarAccess: low track not found");
   }
-  if (lowTrack->first > high) {
+  if (low_track->first > high) {
     return false;
   }
-  auto highTrack = tracks.lower_bound(high);
-  if (highTrack != tracks.end()) {
-    if (highTrack->first > high) {
-      highTrack--;
+  auto high_track = tracks.lower_bound(high);
+  if (high_track != tracks.end()) {
+    if (high_track->first > high) {
+      high_track--;
     }
   } else {
     logger_->error(DRT, 1005, "enclosesPlanarAccess: high track not found");
   }
-  if (highTrack->first - lowTrack->first > (int) layer->getPitch()) {
+  if (high_track->first - low_track->first > (int) layer->getPitch()) {
     return true;
   }
-  if (lowTrack->first - (int) layer->getWidth() / 2 < low) {
+  if (low_track->first - (int) layer->getWidth() / 2 < low) {
     return false;
   }
-  if (highTrack->first + (int) layer->getWidth() / 2 > high) {
+  if (high_track->first + (int) layer->getWidth() / 2 > high) {
     return false;
   }
   return true;
 }
-void FlexPA::prepPoint_pin_genPoints_rect(
-    std::vector<std::unique_ptr<frAccessPoint>>& aps,
-    std::set<std::pair<Point, frLayerNum>>& apset,
-    const gtl::rectangle_data<frCoord>& rect,
-    const frLayerNum layerNum,
-    const bool allowPlanar,
-    const bool allowVia,
-    frAccessPointEnum lowerType,
-    const frAccessPointEnum upperType,
-    const bool isMacroCellPin)
+
+void FlexPA::genAPsFromRect(std::vector<std::unique_ptr<frAccessPoint>>& aps,
```

Comment:
Could you add a short comment that explains roughly what is happening in this method? What does it mean to get access points from rect?